packages/realm/src/app-services/App.ts

```diff
@@ -201,7 +217,7 @@ export class App<FunctionsFactoryType = DefaultFunctionsFactory, CustomDataType
         defaultRequestTimeoutMs: timeout ? binding.Int64.numToInt(timeout) : undefined,
       },
       {
-        baseFilePath: fs.getDefaultDirectoryPath(),
+        baseFilePath: baseFilePath || fs.getDefaultDirectoryPath(),
```

Comment:
Do you actually need to pass `fs.getDefaultDirectoryPath()`? I mean, core knows about this, so doesn't it default to that if `undefined` is passed as `baseFilePath`?

```suggestion
        baseFilePath,
```

Reply:
Actually .. reading the code, I think Core doesn't know about this, in which case I see very little purpose in round-tripping this through C++? Since we don't expose an API to set the default to end-users, I'd suggest treading the "default directory path" as a constant.